crates/ghost_actor/src/ghost_tracker.rs

```diff
@@ -69,14 +70,15 @@ impl<'lt, X: 'lt + Send + Sync, T: 'lt + Send + Sync> GhostTrackerInner<'lt, X,
     /// match up a pending handle request with any pending bookmarks
     fn priv_process_handle(
         &mut self,
+        span: Span,
         user_data: &mut X,
         request_id: RequestId,
         data: T,
     ) -> GhostResult<()> {
         match self.pending.remove(&request_id) {
-            None => return Err(GhostErrorKind::RequestIdNotFound("".to_string()).into()),
+            None => return Err(GhostErrorKind::RequestIdNotFound(String::new()).into()),
             Some(entry) => {
-                (entry.cb)(user_data, Ok(data))?;
+                (entry.cb)(span.child("process_handle"), user_data, Ok(data))?;
```

Comment:
so the thought here is that we use the passed in span rather than the one stored in the entry that we use for the timeout?

This is where I'm a little unsure what the best-practices around spans should be. Do we just let that other one drop? do we put a "fulfilled" message on it?

I think this is totally awesome as a first cut, just curious what we should do next here : )

Reply:
Yeah, I'm not so sure adding a span to GhostTrackerEntry is a good idea so I try not to use it unless I have to.